lambdas/get-addresses/tests/unit/app.test.ts

```diff
@@ -1,26 +1,19 @@
-jest.mock("../../src/services/config-service", () => {
-    return {
-        ConfigService: jest.fn().mockImplementation(() => {
-            return {
-                init: jest.fn().mockResolvedValue([]),
-                config: {
-                    AddressLookupTableName: "Test",
-                },
-            };
-        }),
-    };
-});
-
+jest.mock("@aws-lambda-powertools/parameters/ssm", () => ({
+    getParameter: jest.fn(),
+}));
```

Comment:
This can be moved below the imports - jest automatically hoists mocking calls to the top when running tests.